modules/core/src/v2/coins/eth.ts

```diff
@@ -832,6 +869,48 @@ export class Eth extends BaseCoin {
     }).call(this).asCallback(callback);
   }
 
+  /**
+   * Return types, values, and total amount to send in a batch transaction, using the method signature
+   * `distributeBatch(address[], uint256[])`
+   * @param recipients {EthRecipient[]} transaction recipients
+   * @returns {batchExecutionInfo} the types, values, and total amount to execute the batch transaction
+   */
+  static getBatchExecutionInfo(recipients: EthRecipient[]): batchExecutionInfo {
```

Comment:
make protected if you don't want to. make this part. of the public API

Reply:
Ended up making it protected.